### PR TITLE
Fix upgrade doc to "migrate from yarn"

### PR DESCRIPTION
**Pull request by hzy46**
- change 0.18.0 to 1.0.0
 - change instruction about removing gpu driver

### CODE REVIEW

File: docs/upgrade/upgrade_to_v1.0.0.md [hunk 1]
```diff
@@ -1,21 +1,20 @@
-# Upgrade to v0.18.0
+# Migrate From Yarn Version to v1.0.0
```
Comment (abuccts) [inline docs/upgrade/upgrade_to_v1.0.0.md:1]: YARN

File: docs/upgrade/upgrade_to_v1.0.0.md [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
 
-There are breaking changes between OpenPAI v0.14.0 and v0.18.0. Before v0.18.0, OpenPAI was based on Yarn and Kubernetes, and data was managed by HDFS. Since v0.18.0 ( we might want to change this to v1.0 later after release), OpenPAI has switched to a pure Kubernetes-based architecture. Many new features, such as `AAD authorization`, `Hivedscheduler`, `Kube Runtime`, `Marketplace`, etc., are also included in this release.
+There are breaking changes between OpenPAI v0.14.0 and v1.0.0. Before v1.0.0, OpenPAI was based on Yarn and Kubernetes, and data was managed by HDFS. Since v1.0.0, OpenPAI has switched to a pure Kubernetes-based architecture. Many new features, such as `AAD authorization`, `Hivedscheduler`, `Kube Runtime`, `Marketplace`, etc., are also included in this release.
```
Comment (abuccts) [inline docs/upgrade/upgrade_to_v1.0.0.md:3]: Yarn -> Hadoop YARN

File: docs/upgrade/upgrade_to_v1.0.0.md (hunk 1 above)
Comment (hzy46) [reply docs/upgrade/upgrade_to_v1.0.0.md]: Done.

File: docs/upgrade/upgrade_to_v1.0.0.md (hunk 2 above)
Comment (hzy46) [reply docs/upgrade/upgrade_to_v1.0.0.md]: Done.

**Pull request merged**